### PR TITLE
Feature/92 add playbook action step typ to executor

### PR DESCRIPTION
**Pull request by MaartendeKruijf**

### CODE REVIEW

File: test/unittest/mocks/mock_executor/playbook_action/mock_playbook_action_executor.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package mock_playbook_action_executor
+
+import (
+	"soarca/models/cacao"
+	"soarca/models/execution"
+
+	"github.com/stretchr/testify/mock"
+)
+
+type Mock_Playybook_Action_Executor struct {
```
Comment (hidde-jan) [inline test/unittest/mocks/mock_executor/playbook_action/mock_playbook_action_executor.go:10]:
```suggestion
type Mock_Playbook_Action_Executor struct {
```

Typo.

Also should this be `Mock_PlaybookActionExecutor` to be more consistent with the naming of other mocks?

**Comment by hidde-jan** [comment]
I think this looks pretty good. It's been a while since I looked at the source. Does it already actually execute the playbook-action step with this PR? Or is this just preparation?

**Comment by hidde-jan** [comment]
Oops, I accidentally pulled in the changes from development. Maybe you'd rather rebase instead.

**Comment by MaartendeKruijf** [comment]
@hidde-jan It's final now I've added the call to the executor from the decomposer

**Review by hidde-jan — APPROVED** [review]
Looks very nice

**Pull request merged**